ports/geckolib/gecko_bindings/ptr.rs

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use bindings::*;
+use heapsize::HeapSizeOf;
+use std::fmt::{self, Debug};
+
+macro_rules! define_holder_gecko_arc {
```

Comment:
I'd also call this define_gecko_holder_arc or just define_holder_arc since we're in geckolib.